drake/multibody/rigid_body_plant/visualization/README

```diff
@@ -2,3 +2,8 @@ To run drake-visualizer with the contact force visualization:
 
 $ cd drake-distro
 $ ./build/install/bin/drake-visualizer --script ./drake/multibody/rigid_body_plant/visualization/contact_viz.py
+
+To run drake-visualizer with the simulation and real time factor display:
+
+$ cd drake-distro
+$ ./build/install/bin/drake-visualizer --script ./drake/multibody/rigid_body_plant/visualization/show_time.py
```

Comment:
rather than make this a script, make it a module in director and add it to the Tools menu of the drake-visualizer app (perhaps off by default).  You use the Tools menu in the past, I believe, when you implemented the contact visualizer.  Note, this will require a commit to director which I will have to review.